usaspending_api/disaster/v2/views/cfda/count.py

```diff
@@ -0,0 +1,34 @@
+from elasticsearch_dsl import Q as ES_Q
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from usaspending_api.common.cache_decorator import cache_response
+from usaspending_api.common.query_with_filters import QueryWithFilters
+from usaspending_api.disaster.v2.views.disaster_base import DisasterBase
+from usaspending_api.search.v2.elasticsearch_helper import get_number_of_unique_terms_for_awards
+
+
+class CfdaCountViewSet(DisasterBase):
+    """
+    This route takes DEF Codes and Award Type Codes and returns count of CFDA.
+    """
+
+    endpoint_doc = "usaspending_api/api_contracts/contracts/v2/disaster/cfda/count.md"
+
+    required_filters = ["def_codes", "award_type_codes"]
+
+    @cache_response()
+    def post(self, request: Request) -> Response:
+        filter_query = QueryWithFilters.generate_awards_elasticsearch_query(self.filters)
+
+        # Ensure that only non-zero values are taken into consideration
+        non_zero_columns = ["total_covid_obligation", "total_covid_outlay", "face_value_loan_guarentee"]
```

Comment:
Typo here. I believe it's `face_value_loan_guarantee` in ES and everywhere else.

But also, I thought we were supposed to be using `total_loan_value` for loan aggregations, which in the API has been referred to as "face_value_of_loan" (no guarantee).

Reply:
The `ElasticsearchLoansPaginationMixin` seems to have it right in its `sum_column_mapping` field.

https://github.com/fedspendingtransparency/usaspending-api/pull/2573/files#diff-5a427bd0e842bfcd87ac12ab1e696f58R237

Reply:
Will make sure to fix this. I originally was using `face_value_loan_guarentee` everywhere until I realized it needed to be `total_loan_value`. Apparently I missed updating here.

As for the `sum_column_mapping` it is, but I did not want to use that mixin here since it adds a Pagination property that doesn't make sense for a count endpoint.